README.md

```diff
@@ -47,13 +66,35 @@ Update your scripts object in `package.json` to include the following
 }
 ```
 
+Optional, add ` && pretty-quick` or your own prettifier command to the script to post-process the schema code. 
+
 ## Running the script
 
 Run `npm run generate-schema` to re-generate your utility functions every time your airtable base updates!
 
-## Optional Inputs
+## Important Assumptions
```

Comment:
Does this still assume the Airtable API key is stored as `process.env.REACT_APP_AIRTABLE_API_KEY`? I have to change that line every time haha